signin.html

```diff
@@ -1,46 +1,53 @@
-<!DOCTYPE html>
+<!doctype html>
 <html>
   <head>
     <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>로그인</title>
     <link
       rel="stylesheet"
       type="text/css"
       href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard/dist/web/static/pretendard.css"
     />
-    
     <link rel="stylesheet" href="./src/reset.css" />
     <link rel="stylesheet" href="./src/sign-style.css" />
- 
+  </head>
   <body>
     <a href="/">
-      <img src="/images/logo.svg" class="logo"> 
+      <img src="/images/logo.svg" class="logo" />
     </a>
-    
     <div class="ask-signup">
       회원이 아니신가요?
       <a href="/signup.html" class="signup">회원 가입하기</a>
     </div>
-
-    <form>
-      <label for="email" class="email-label">이메일</label>
-      <input type="email" id="email" class="email-input">
-      <label for="password" class="password-label">비밀번호</label>
-      <input type="password" id="password" class="password-input">
-      <button type="submit" class="button">로그인</button>
-    </form>
-
-    <div class="sns-signin">
-      <div class="sns-text">소셜 로그인</div>
-      <div class="sns-logo">
-        <div class="google-block">
-          <a href="https://www.google.com/" class="google-logo"><img src="/images/google.png" alt="구글 로고" ></a>
+    <container class="container">
```

Comment:
없는 HTML 태그를 사용해주신 것 같아요. 
존재하지 않는 태그명을 사용했을 때 브라우저의 기본 동작이 어떻게 되는지 알아보아도 좋겠어요 🚀

```suggestion
    <div class="container">
```